wpilibc/src/main/native/include/frc/geometry/Twist2d.h

```diff
@@ -31,5 +31,25 @@ struct Twist2d {
    * Angular "dtheta" component (radians)
    */
   units::radian_t dtheta = 0_rad;
+
+  /**
+   * Checks equality between this Twist2d and another object.
+   *
+   * @param other The other object.
+   * @return Whether the two objects are equal.
+   */
+  bool operator==(const Twist2d& other) const {
+    return units::math::abs(dx - other.dx) < 1E-9_m &&
+           units::math::abs(dy - other.dy) < 1E-9_m &&
+           units::math::abs(dtheta - other.dtheta) < 1E-9_rad;
```

Comment:
Is there a reason this one is inline while the others are not?

Reply:
Since Twist2d only has two small functions, namely these two, it's appealing to keep it header-only so everything's in one place and easy to read. I don't feel strongly about it though.

Reply:
^ yeah, that was the reasoning behind it